test/archethic/transaction_chain/transaction_summary_test.exs

```diff
@@ -3,5 +3,39 @@ defmodule Archethic.TransactionChain.TransactionSummaryTest do
 
   alias Archethic.TransactionChain.TransactionSummary
 
```

Comment:
You may add test for the 2 new functions of TransactionSummary (resolve_addresses and equals?)